src/logic/safe/utils/spendingLimits.ts

```diff
@@ -157,7 +157,7 @@ export const enableSpendingLimitModuleMultiSendTx = (safeAddress: string): Multi
     to: multiSendTx.to,
     value: Number(multiSendTx.valueInWei),
     data: multiSendTx.txData as string,
-    operation: DELEGATE_CALL,
+    operation: CALL,
```

Comment:
Call is the default value, maybe we don't really need to pass it

Reply:
It's required by the type.

Reply:
In my opinion we should remove `operation` from the `MultiSigTx` type. For two reasons:

1. `getEncodedMultiSendCallData` ignores it anyways
2. We should never use something else than `CALL` anyways.

Alternatively `getEncodedMultiSendCallData` could `throw` if the type is not `CALL`.

Reply:
I'll adjust it 👌